### ISSUE TITLE
Issues related to labels

### ISSUE THREAD

**Issue by juzen2003** [SETI/pds-hst-pipeline#53]
Current issues related to labels: (will continue to update this list)
- Document dir:
    - In document label, under identification area, how do we define version id? Currently, we are checking proposal files in backups and bump the minor version number
    - In document label, under docment edition area, how do we define edition name?
    - In document collection label, under modification history, how do we define version id?
- Schema dir:
    - Need to figure out the inventory (& its version) for schema csv & label
    - In schema collection label, under modification history, how do we define version id? Currently it's 1.0.

**Comment by matthewtiscareno** [comment]
>       * In document label, under identification area, how do we define version id? Currently, we are checking proposal files in backups and bump the minor version number

The first version we create of any product is v1.0.  If we update the content of the product itself, we increment the first digit.  If we update the label at a later time, we increment the second digit.

Note that it doesn't matter whether this product has had 40 different versions in MAST in the past.  The first version to go into PDS4 is v1.0.

Note that it also doesn't matter whether we have gone through a bunch of versions as we develop this code.  It is the first version to be officially archived in PDS4 that is v1.0.

>       * In document collection label, under modification history, how do we define version id?

Same as above.  If the collection inventory has changed, then increment the first digit.  If only the label describing the collection inventory has changed, then increment the second digit.

**Comment by matthewtiscareno** [comment]
> In document label, under docment edition area, how do we define edition name?

Looking at the Cassini ISS bundle, it seems that we have used `<edition_name>First</edition_name>`

Hey @mace-space, do you have any guidance about when we should do any differently?

**Comment by matthewtiscareno** [comment]
These also seem like questions for @mace-space, perhaps with help from @esimpsons3ti, @mit3ch, @mseritan.

> Schema dir:
> Need to figure out the inventory (& its version) for schema csv & label
> In schema collection label, under modification history, how do we define version id? Currently it's 1.0.

The `collection_schema` currently includes entries for the PDS, HST, and DISP dictionaries.  Do we just need to make sure it includes the version of each dictionary that we've used to create the bundle?  This may not actually be the latest version, right?

> Context dir:
> Need to determine the version id for each entry in the csv

Actually, do we want the `collection_context` inventory to include LIDVIDs?  Maybe we only want LIDs, so that people will always get the latest version?

**Comment by matthewtiscareno** [comment]
> Context dir:
> In collection context label, under modification history, how do we define version id? Currently it's 1.0.
> In investigation label, under modification history, how do we define version id? Currently it's 1.0.

Yes, these should be v1.0, because the version that eventually gets archived will be the first one to ever go into PDS4.

> Do we move or copy files from staging folder?

When we're ready to complete this project, we'll work together on how to deploy and how to clean up.

> Product dir:
> How do we name the browse product directory? browse_{inst_id}_suffix and data_{inst_id}_suffix? Collection_name only returns either data_{inst_id}_suffix or miscellaneous_{inst_id}_suffix

Yes, I should think the browse directories will follow the same format as the data and miscellaneous directories.

> Need to determine the version id for each product file, put 1.0 for now in csv
> In collection label, need to determine the version id
> How do we determine the version of the bundle label?

Again, the first version of each file to get archived at the end of this project will be v1.0.

> will there be multiple channel ids in one collection product? Now we assume only all products with the same collection name will have one channel id

@juzen2003 does not remember what this was about.

> In collection label, reference list area, do we only include both handbooks?

That's fine for now.  We can revise if we decide to do so.

> Bundle label:
> DOI value

We will work together to obtain the DOI when the time comes.  For now, use a dummy value such as `<doi>10.xxxxx/xxxxxxx</doi>`
